source/retryable-writes/retryable-writes.rst

```diff
@@ -52,65 +51,57 @@ Transaction ID
-   fields. The ``sessionId`` component is the corresponding server session ID,
-   which is an opaque BSON value returned by the ``startSession`` command. The
-   ``txnNum`` component is a monotonically increasing (per server session),
+   fields. The ``sessionId`` component is the corresponding server session ID.
+   which is an opaque BSON value defined in the Driver Session specification.
+   The ``txnNum`` component is a monotonically increasing (per server session),
    positive 64-bit integer.
 
 DriverSession
-   Driver object representing a client session and the operations that can be
-   performed on it, as defined in the Driver Session specification. This object
-   is always associated with a server session. The name of this object MAY vary
-   across drivers.
-
-SessionOptions
-   Data structure defined in the Driver Session specification to provide options
-   when creating a DriverSession. This could be a strongly typed class in some
-   drivers, or it could be a loosely typed dictionary in other drivers.
+   Driver object representing a client session, which is defined in the Driver
+   Session specification. This object is always associated with a server
+   session; however, drivers will pool server sessions so that creating a
+   DriverSession will not always entail creation of a new server session. The
+   name of this object MAY vary across drivers.
 
 Additional terms may be defined in the Driver Session specification.
 
 Naming Deviations
 -----------------
 
-This specification defines the name for a new DriverSession option,
-``retryWrites``. To the extent possible, drivers SHOULD use the defined name.
-However, where a driver’s or language’s naming conventions would conflict,
-drivers SHOULD honor their existing conventions. For example, a driver may use
-``retry_writes`` instead of ``retryWrites``.
+This specification defines the name for a new MongoClient option,
+``retryWrites``. Drivers MUST use the defined name to ensure portability of
+connection strings across applications and drivers.
 
-Configuration
--------------
+MongoClient Configuration
+-------------------------
 
-The following option MUST be added to the SessionOptions data structure:
+This specification introduces the following client-level configuration option.
 
-.. code:: typescript
+retryWrites
+~~~~~~~~~~~
 
-  class SessionOptions {
-    /**
-     * If true, retryable behavior will be applied to write commands
-     * executed within the session.
-     *
-     * Defaults to false, which implies no change in write behavior.
-     */
-    retryWrites: Optional<Boolean>;
-  }
+This boolean option determines whether retryable behavior will be applied to all
+write operations executed within the application. This option MUST default to
+false, which implies no change in write behavior.
+
+This option MUST NOT be configurable at the level of a database object,
+collection object, or at the level of an individual write operation.
```

Comment:
The take-away from our previous spec review meeting was to only have it as a client-level option. I'd like to be conservative here and avoid having drivers add the option everywhere, even if there is no technical limitation against doing so. A limited introduction also reduces the burden of API changes and documentation.